src/DssSpell.sol

```diff
@@ -49,82 +127,196 @@ contract DssSpellAction is DssAction {
     //    https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
     //
 
-    // --- Rates ---
-    // uint256 internal constant SEVEN_PT_FIVE_PERCENT_RATE = 1000000002293273137447730714;
+    function actions() public override {
+        // ----------------- Compound v2 D3M Onboarding -----------------
+        // https://vote.makerdao.com/polling/QmWYfgY2#poll-detail
```

Comment:
Is there a link for this for a forum post yet?
Only 3 tagged forum posts and none of them are official announcements of the work; perhaps we can use the signal request? 
https://forum.makerdao.com/tag/compound-d3m

Reply:
There is no hard requirement for both a forum post and vote. We just need to reference whatever is put forward by governance.